cerberus/kubernetes/client.py

```diff
@@ -78,6 +78,32 @@ def list_namespaces():
     return namespaces
 
 
+# Monitor the status of all specified namespaces
+# and set the status to true or false
+def monitor_namespaces_status(watch_namespaces, watch_terminating_namespaces, iteration, iter_track_time):
+    namespaces = []
+    none_terminating = True
+    if watch_terminating_namespaces:
+        watch_nodes_start_time = time.time()
+        try:
+            ret = cli.list_namespace(pretty=True)
+        except ApiException as e:
+            logging.error("Exception when calling CoreV1Api->list_namespaced_pod: %s\n" % e)
```

Comment:
Maybe sys.exit(1) to handle this case? Thoughts?

Also s/list_namespaced_pod/list_namespace.